app/views/volunteers/show.html.slim

```diff
@@ -174,6 +174,12 @@ h3= t('.availability')
       h3= t('assignments')
   = render 'assignments/volunteer_index', assignments: @volunteer.assignments
 
+- if @volunteer.group_assignments.any?
+  .row
+    .col-xs-12#group_assignments
+      h3= t('.group_assignments')
```

Comment:
why nest h3? 
It could be just `h3#group_assignments`. The `.row` and `.col-xs-12` are just bloating the html and have no use.